main.go

```diff
@@ -157,7 +157,7 @@ func runLoop(ctx context.Context) {
 
 	go func() {
 		for {
-			config.LogFeatures()
+			//config.LogFeatures()
```

Comment:
minor: can you delete this instead of leaving it commented out?

Reply:
done